infrastructure/aws/variables.tf

```diff
@@ -502,3 +502,10 @@ variable "django_queue_max_attempts" {
   default     = 1
   description = "How many attempts to run unstructured task"
 }
+
+variable "embedding_backend" {
+  type        = string
+  default     = "text-embedding-ada-002"
```

Comment:
just to be safe should we set this to our current model? text-embedding-3-large?

Reply:
agreed